Notepad-siddhantkhariwal/Readme.md

```diff
@@ -0,0 +1,10 @@
+*********Notepad***********
```

Comment:
Update Readme as per contribution guidelines :
- Name of Project
- Demo of Project in .gif how your project is look 
- Description of project 
-  Requirements or Setup instructions
-  Name of Author(s)  
- [Demo Readme](https://github.com/Tenet-Coding/Hacktoberfest-Projects/blob/master/ConwaysGameOfLife/README.md)